frontend/public/plugins.ts

```diff
@@ -1,6 +1,21 @@
+import * as _ from 'lodash-es';
 import { PluginStore } from '@console/plugin-sdk/src/store';
 import { ActivePlugin } from '@console/plugin-sdk/src/typings';
-import { loadPluginFromURL } from '@console/dynamic-plugin-sdk/src/runtime/plugin-loader';
+import { getURLSearchParams } from './components/utils/link';
+
+const getAllowedDynamicPluginNames = () => {
```

Comment:
nit: Should this be `getEnabledDynamicPluginNames`?

Reply:
Sounds good, will update.